homeassistant/components/advantage_air/config_flow.py

```diff
@@ -25,7 +26,7 @@ class AdvantageAirConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     DOMAIN = DOMAIN
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(self, user_input=None) -> FlowResult:
```

Comment:
```suggestion
    async def async_step_user(self, user_input: dict[str, Any] | None = None) -> FlowResult:
```

Reply:
This change breaks it in my dev environment, despite seeing this exact same pattern in other integrations.

> unsupported operand type(s) for |
Alternative syntax for unions requires Python 3.10 or newer

Reply:
You need to add `from __future__ import annotations` at the top to be able to use `X | Y` syntax.

Reply:
I think you lost this in a rebase... it is still missing.

Reply:
Oops, forgot to add it to the commit, adding it now.